apps/deltalake-glow/DeltalakeGlow.hs

```diff
@@ -0,0 +1,87 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE DataKinds #-}
+
+module Main where
+
+import Control.Distributed.Spark
+import Control.Distributed.Spark.SQL.Dataset as Dataset
+import Data.Int (Int32)
+
+
+main :: IO ()
+main = forwardUnhandledExceptionsToSpark $ do
+    conf <- newSparkConf "Sparkle Dataset demo"
+    confSet conf "spark.jars.packages" "io.delta:delta-core_2.11:0.4.0,io.projectglow:glow-spark2_2.11:1.1.2"
+    confSet conf "spark.hadoop.io.compression.codecs" "io.projectglow.sql.util.BGZFCodec"
+    confSet conf "spark.sql.extensions" "io.delta.sql.DeltaSparkSessionExtension"
+    confSet conf "spark.sql.catalog.spark_catalog" "org.apache.spark.sql.delta.catalog.DeltaCatalog"
+
+    session <- do { sess <- builder >>= (`config` conf) >>= getOrCreate
+                  ; session <- registerGlow sess >>= registerUDFDenseMatrix
+                  ; return session
```

Comment:
```suggestion
                  ; registerGlow sess >>= registerUDFDenseMatrix
```
This simplification applies in other places of this file too.

Reply:
I'll have them modified :rocket: